### PR TITLE
Fix lint commands and resolve lint issues

### PR DESCRIPTION
**Pull request by jcfilben**

### CODE REVIEW

File: grommet-leaflet/package.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
   "license": "ISC",
   "main": "src/index.js",
   "scripts": {
-    "lint-fix": "eslint src --fix"
+    "lint": "eslint src/**/*.js src/**/*.jsx",
```
Comment (taysea) [inline grommet-leaflet/package.json:13]: In grommet, we just have `eslint src` -- is there something we can add to our eslintrc to have it lint jsx pages too?

Comment (jcfilben) [reply grommet-leaflet/package.json]: I couldn;t find anything that we could add to the eslintrc file, but I did find a better way to write the command in the package.json files

File: .eslintrc.js [hunk 2]
```diff
@@ -72,6 +75,6 @@ module.exports = {
     'import/no-dynamic-require': 0,
     quotes: [2, 'single', { avoidEscape: true }],
     'arrow-body-style': 0,
-    allowElseIf: true,
```
Comment (taysea) [inline .eslintrc.js:75]: Just confirming this diff is desired?

Comment (jcfilben) [reply .eslintrc.js:75]: Yes, it seemed like eslint didn't recognize the `allowElseIf` rule and I think the equivalent is setting `no-else-return: 0`

**Review by taysea — APPROVED** [review]

**Pull request merged**